libs/utils/src/filenames.ts

```diff
@@ -132,6 +132,9 @@ export function generateCastVoteRecordReportFilename(
 }
 
 /* Extract information about a CVR file from the filename */
+// Expected filename format with human-readable separators is:
+// [TEST__]machine_{machineId}__{numberOfBallots}_ballots__YYYY-MM-DD_HH-mm-ss.jsonl
+// This format is current as of 2023-02-22 and may be out of date if separator constants have changed
```

Comment:
Thanks for updating this. Will be killing this exact format but will be using something similar.